README.arm.md

```diff
@@ -1,32 +1,24 @@
-# Julia binaries for ARM
+# Julia on ARM (Linux)
 
-[Nightly builds](https://status.julialang.org/download/linux-arm) are
-available for ARMv7-A.
-
-# Hardware requirements
+Julia fully supports ARMv8 (AArch64) processors, and supports ARMv7 and ARMv6
+(AArch32) with some caveats. This file provides general guidelines for compilation,
+in addition to instructions for specific devices.
 
-Julia requires at least `armv6` and `vfpv2` instruction sets. It's recommended
-to use at least `armv7-a`. `armv5` or soft float are not supported.
+A list of [known issues](https://github.com/JuliaLang/julia/labels/arm) for ARM is
+available. If you encounter difficulties, please create an issue including the output
+from `cat /proc/cpuinfo`.
 
-# Building Julia on ARM
 
-Julia has been compiled on several ARMv7 / Cortex A15 Samsung
-Chromebooks running Ubuntu Linux under Crouton, Raspberry Pi systems
-and Odroid boards. This is a work in progress - several tests are
-known to fail, and backtraces are not available.
+## Building Julia
 
-Julia on ARM can be built by simply typing `make`, which will download all
-the relevant libraries. This is the *recommended* way, and it will take a
-few hours.
+Julia has been successfully compiled on several ARMv7 / Cortex A15 Samsung Chromebooks
+running Ubuntu Linux under Crouton, a number of Raspberry Pi variants, Odroid boards,
+and the nVidia Jetson TX2.
```

Comment:
Doesn't have to be included here (since not everyone has access to these system) but the aarch64 systems I've tested include Jetson TX1 (the first successful build), X-Gene1, Overdrive 3000 (buildbot), Cavium ThunderX (from packet.net).